pwndbg/commands/context.py

```diff
@@ -204,6 +204,9 @@ def __eq__(self, other):
     def write(self, data) -> None:
         self.func(data)
 
+    def writelines(self, lines_iterable) -> None:
+        self.func("".join(lines_iterable))
```

Comment:
Wouldnt it be better to call func multiple times? Tbh no idea, this is probably better, huh

Reply:
I tried that first but there is no way to know when a new list of outputs is written or if the current write is part of the same output.

The TUI windows are cleared before the new output is written from a call to `write` to always show the output at the same position in the window. This caused only the last line from `writelines` to be displayed. We'd need to add some other event or heuristic to clear the output buffer between context displays if we'd want to allow incremental writes with the `CallOutput` redirection method. And just writing all lines at once was the easier solution to this bug.